dynawo/sources/Models/CPP/ModelNetwork/DYNModelRatioTapChanger.cpp

```diff
@@ -54,31 +54,22 @@ ModelRatioTapChanger::getUpIncreaseTargetU() {
     increaseRate = (steps_[lowStepIndex_].getRho() < steps_[lowStepIndex_ + 1].getRho());
 
   // in tfo, the main equation is U2 = rho*U1
-  // if V1 > V2, then the target is V2
-  // when rho increase U2 increase
-  // if V2 > V1, the target is V1; V1 = U2/rho
-  // when rho increase V1 decrease
-  if (V1SupV2_) {
+  if (side_ == "ONE") {
     if (increaseRate) {
-      upIncreaseTargetU = true;
-    } else {
       upIncreaseTargetU = false;
+    } else {
+      upIncreaseTargetU = true;
     }
-  } else {
+  } else if (side_ == "TWO") {
     if (increaseRate) {
-      upIncreaseTargetU = false;
-    } else {
       upIncreaseTargetU = true;
+    } else {
+      upIncreaseTargetU = false;
     }
```

Comment:
What if terminalRef not present into iidm file? Shouldn't there be an else that throw an error?

Reply:
if terminalRef is not present in the iidm file, no taps are changed

Reply:
see #242